src/poliastro/czml/extract_czml.py

```diff
@@ -494,5 +494,158 @@ def add_orbit(
 
         self.i += 1
 
+    def add_trajectory(
+        self,
+        positions,
+        epochs,
+        groundtrack_show=False,
+        groundtrack_lead_time=None,
+        groundtrack_trail_time=None,
+        groundtrack_width=None,
+        groundtrack_color=None,
+        id_name=None,
+        id_description=None,
+        path_width=None,
+        path_show=None,
+        path_color=None,
+        label_fill_color=None,
+        label_outline_color=None,
+        label_font=None,
+        label_text=None,
+        label_show=None,
+    ):
+        """
+        Adds trajectory
+
+        Parameters
+        ----------
+        positions: ~astropy.coordinates.CartesianRepresentation
+            Trajectory to plot.
+        epochs: ~astropy.time.core.Time
+            Epochs for positions.
+        Groundtrack parameters:
```

Comment:
Let's remove these intermediate headers, they are not rendered correctly in the documentation.